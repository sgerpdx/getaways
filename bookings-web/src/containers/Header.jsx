import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => {
  return (
    <>
      <h4>Home</h4>
      <hr></hr>
    </>
  );
};

export default Header;
