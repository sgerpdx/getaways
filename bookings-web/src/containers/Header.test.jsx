require('dotenv').config();
import React from 'react';
import { screen, render, waitFor } from '@testing-library/react';
// import userEvent from '@testing-library/user-event';
// import { rest } from 'msw';
// import { setupServer } from 'msw/node';
import Header from './Header';
import { MemoryRouter, Route } from 'react-router-dom';
//jest.useFakeTimers();

describe('Header container component', () => {
  it('displays the correct title text and nav controls', async () => {
    render(
      <MemoryRouter>
        <Route path="/">
          <Header />
        </Route>
      </MemoryRouter>
    );

    const heading = await screen.getByText('Home');
    expect(heading).toMatchSnapshot();
  });
});
