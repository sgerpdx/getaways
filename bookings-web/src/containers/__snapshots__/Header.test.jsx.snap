// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header container component displays the correct title text and nav controls 1`] = `
<h4>
  Home
</h4>
`;

exports[`Header container component displays the correct title text and nav controls 2`] = `Promise {}`;
